app/main.ipynb

```diff
@@ -58,17 +228,22 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 1,
-   "outputs": [],
    "source": [
-    "# write your code for task 1 here"
+    "male_amount = (adults_df[\"sex\"] == \"Male\").sum()\n",
+    "female_amount = (adults_df[\"sex\"] == \"Female\").sum()"
    ],
    "metadata": {
     "collapsed": false,
     "pycharm": {
      "name": "#%%\n"
+    },
+    "ExecuteTime": {
+     "end_time": "2024-04-30T02:51:26.463634Z",
+     "start_time": "2024-04-30T02:51:26.420401Z"
     }
-   }
+   },
+   "outputs": [],
```

Comment:
For all tasks besides 0 and 9 there aren't any output results shown

Reply:
<img width="948" alt="image" src="https://github.com/mate-academy/py-adult-data-analysis/assets/61380547/a38991eb-855b-4a0d-8031-d8b9744033bd">